src/DssSpell.sol

```diff
@@ -124,146 +62,127 @@ contract DssSpellAction is DssAction {
-        oracles[4] = 0x16655369Eb59F3e1cAFBCfAC6D3Dd4001328f747;
-        oracles[5] = 0xC9508E9E3Ccf319F5333A5B8c825418ABeC688BA;
-        oracles[6] = 0xA8EB82456ed9bAE55841529888cDE9152468635A;
-        oracles[7] = 0x83e23C207a67a9f9cB680ce84869B91473403e7d;
-        oracles[8] = 0xDA1d2961Da837891f43235FddF66BAD26f41368b;
-        oracles[9] = 0x4b0E327C08e23dD08cb87Ec994915a5375619aa2;
-        oracles[10] = 0xfeEd00AA3F0845AFE52Df9ECFE372549B74C69D2;
-        oracles[11] = 0x8aFBD9c3D794eD8DF903b3468f4c4Ea85be953FB;
-        oracles[12] = 0x8de9c5F1AC1D4d02bbfC25fD178f5DAA4D5B26dC;
-        oracles[13] = 0xd94BBe83b4a68940839cD151478852d16B3eF891;
-        oracles[14] = 0xa580BBCB1Cee2BCec4De2Ea870D20a12A964819e;
-        oracles[15] = 0x75ef8432566A79C86BBF207A47df3963B8Cf0753;
-        oracles[16] = 0xD27Fa2361bC2CfB9A591fb289244C538E190684B;
-        oracles[17] = 0x60da93D9903cb7d3eD450D4F81D402f7C4F71dd9;
-        oracles[18] = 0x71eCFF5261bAA115dcB1D9335c88678324b8A987;
-        oracles[19] = 0x77EB6CF8d732fe4D92c427fCdd83142DB3B742f7;
-        oracles[20] = 0x8ff6a38A1CD6a42cAac45F08eB0c802253f68dfD;
-        oracles[21] = 0x130431b4560Cd1d74A990AE86C337a33171FF3c6;
-        oracles[22] = 0x3CB645a8f10Fb7B0721eaBaE958F77a878441Cb9;
-        oracles[23] = 0xd72BA9402E9f3Ff01959D6c841DDD13615FFff42;
-        TeleportOracleAuthLike(ORACLE_AUTH).addSigners(oracles);
-
-        // Configure TeleportRouter
-        TeleportRouterLike(ROUTER).rely(esm);
-
-        TeleportRouterLike(ROUTER).file("gateway", DOMAIN_ETH, TELEPORT_JOIN);
-        TeleportRouterLike(ROUTER).file("gateway", DOMAIN_OPT, TELEPORT_GATEWAY_OPT);
-        TeleportRouterLike(ROUTER).file("gateway", DOMAIN_ARB, TELEPORT_GATEWAY_ARB);
-
-        // Authorize TeleportGateways to use the escrows
-        EscrowLike(escrowOpt).approve(dai, TELEPORT_GATEWAY_OPT, type(uint256).max);
-        EscrowLike(escrowArb).approve(dai, TELEPORT_GATEWAY_ARB, type(uint256).max);
-
-        // Add to ilk registry
-        ilkRegistry.put(
-            ILK,
-            TELEPORT_JOIN,
-            address(0),
-            0,
-            4,
-            address(0),
-            address(0),
-            "",
-            ""
+        // ----------------------- MKR Vesting Transfers -----------------------
+        // RWF-001 - 38 MKR - 0x96d7b01Cc25B141520C717fa369844d34FF116ec
+        GemLike(DssExecLib.mkr()).transfer(RWF_WALLET, 38 * WAD);
+
+        // DECO-001 - 125 MKR - 0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7
+        GemLike(DssExecLib.mkr()).transfer(DECO_WALLET, 125 * WAD);
+
+        // ------------------------ MKR Vesting Stream -------------------------
+        // GRO-001 | 2022-07-01 to 2023-07-01 | 803 MKR | 0x7800C137A645c07132886539217ce192b9F0528e Cliff Date = 2022-07-01
+        MCD_VEST_MKR_TREASURY.restrict(
+           MCD_VEST_MKR_TREASURY.create({
+                _usr: GRO_WALLET,
+                _tot: 803 * WAD,
```

Comment:
`gov` allowance should be increased via `gov.approve` by the `tot` amount of the new stream, following previous patterns, including also coverage for the `allowance` in specific tests.

Reply:
Nice catch.
As discussed this was missing in at least 3 spells since migration was done to the treasury vesting contract.
Will add for the current stream, for the past spell let's do it separately in one of the upcoming spells.

Reply:
[f1cdbec](https://github.com/makerdao/spells-mainnet/pull/270/commits/f1cdbec5008112f33b56e01a7561f705e9659309)